serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/deserializers/protobuf/ProtobufSchemaParser.java

```diff
@@ -3,6 +3,7 @@
 import com.google.protobuf.Descriptors;
 import com.squareup.wire.schema.internal.parser.ProtoFileElement;
 import com.squareup.wire.schema.internal.parser.ProtoParser;
+//import com.amazonaws.services.schemaregistry.utils.apicurio.FileDescriptorUtils;
```

Comment:
Are we using the patched FileDescriptorUtils instead of the actual FileDescriptorUtils?

Reply:
Fixed, patched in our own